owa/README.md

```diff
@@ -0,0 +1,47 @@
+# Bed Management OWA
+> Bed management OpenMRS OWA is a application for managing admission locations (Add/Edit/Delete/listing wards/beds).
+
+## Development
+#### Local Setup Instructions
+1. Move into the owa directory
+```bash
+cd openmrs-module-bedmanagement/owa
+```
+
+2. Install the dependencies
+```bash
+$ npm install
+
+# Or you can use yarn if you have installed yarn
+$ yarn install
+```
+
+3. Build & compile code
+```bash
+# npm command
+$ npm run webpack
+$ npm run build-webpack # pack with Compress for production
+
+# yarn command
+$ yarn webpack
```

Comment:
@sanishmaharjan and then, where do we see the app for debugging?

Reply:
I use Chrome React Developer Tool for debugging.

Reply:
I mean, what's the URL to go to?

Reply:
- https://192.168.33.10/owa/bedmanagement/admissionLocations.html
Since I have set 
App Base URL: https://192.168.33.10/owa/

Reply:
No I mean the standalone URL to view the React app. Or then why would anyone run `yarn webpack`, what does this actually do?

Reply:
"yarn webpack" command bundle reactJs code and watch it change so that code changes will reflect.